benchmark/src/real_datast.R

```diff
@@ -0,0 +1,42 @@
+source("src/transformations/transformation_helper.R")
+file = readRDS("realdataset,rds") #require both count matrix and size factor vector
```

Comment:
This looks like a typo. I assume you mean `"realdataset.rds"`, right?

Reply:
Thanks for your comments. I have fixed it.